### PR TITLE
README instructions update

### PR DESCRIPTION
**Pull request by ghost**
Refered to async/await refactoring ref #17

### CODE REVIEW

**Comment by lirantal** [comment]
someone's on fire!

File: README.md [hunk 1]
```diff
@@ -47,20 +47,14 @@ After the OO module is initialized with the correct options, we can begin queryi
 ## Statistics
 The following example will query for the statistics, and you can inspect the `err` or `data` object for the returned result.
 ```javascript
-OO.dashboard.statistics(function(err, data) {
-	console.log(err);
-	console.log(data);
-});
+const data = await OO.dashboard.statistics(); // Throws err if Promise got rejected
```
Comment (lirantal) [inline README.md]: maybe we should wrap it with try/catch as a good example?

Comment (lirantal) [reply README.md]: and same for the configuration example below it

Comment (ghost) [reply README.md]: Okay :+1:

**Comment by ghost** [comment]
Done

**Pull request merged**